components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -1186,4 +1189,19 @@ private boolean validateUpdateOfIdPEntityId(FederatedAuthenticatorConfig[] curre
         return true;
     }
 
+    private String getResidentIdPEntityId() {
+
+        if (residentIdPEntityId == null) {
+            synchronized (IdentityProviderManager.class) {
+                if (residentIdPEntityId == null) {
+                    residentIdPEntityId = IdentityUtil.getProperty("SSOService.EntityId");
```

Comment:
This is not applicable to the current code, since the PR is closed. 

Current code uses this defined constant : https://github.com/wso2/carbon-identity/blob/master/components/identity-core/org.wso2.carbon.identity.base/src/main/java/org/wso2/carbon/identity/base/IdentityConstants.java#L176